### PR TITLE
[WebGL2] revise softmax

### PR DESCRIPTION
**Pull request by GreyZzzzzzXh**
Fix bug #90 .

### CODE REVIEW

File: src/nn/webgl2/webgl/fragmentShader/activation/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { RELU } from './RELU'
 import { RELU1 } from './RELU1'
 import { RELU6 } from './RELU6'
-import { softmax } from './softmax'
 import { fuse } from './fuse'
+import softmax from './softmax'
```
Comment (huningxin) [inline src/nn/webgl2/webgl/fragmentShader/activation/index.js]: Why do we need to change this line?

**Comment by huningxin** [comment]
@GreyZzzzzzXh , thanks for the PR!

In this PR, you have two commits with same name. Could you please squash them into one? And please also address my comment inline. After that, we can merge it.

File: src/nn/webgl2/webgl/fragmentShader/activation/index.js (hunk 1 above)
Comment (GreyZzzzzzXh) [reply src/nn/webgl2/webgl/fragmentShader/activation/index.js]: This is used to import default export.

**Comment by GreyZzzzzzXh** [comment]
> Could you please squash them into one?

done.

**Comment by huningxin** [comment]
Looks good. Merge.

**Pull request merged**